domain/src/Aggregates/TaxYear/ValueObjects/CapitalGain.php

```diff
@@ -20,11 +20,6 @@ public function __construct(
         $this->difference = $proceeds->minus($costBasis);
     }
 
-    public function opposite(): self
-    {
-        return new self($this->costBasis->opposite(), $this->proceeds->opposite());
-    }
```

Comment:
ℹ️ Not used anymore (nor is the `opposite` method of `FiatAmount` and `Quantity`)